src/RpcClient.ts

```diff
@@ -506,96 +506,137 @@ export class RpcClient {
    * Build and send an optimized transaction, and handle its confirmation status
    * @param {TransactionInstruction[]} instructions - The transaction instructions
    * @param {Keypair} fromKeypair - The sender's keypair
-   * @param {boolean} skipPreflightChecks - Whether the transaction should skip preflight checks. Defaults to `true`
-   * @param {number} maxRetries - The maximum number of times to retry sending the transaction to the leader. Defaults to 0
+   * @param {AddressLookupTableAccount[]} lookupTables - The lookup tables to be included in a versioned transaction. Defaults to `[]`
+   * @param {boolean} skipPreflightChecks - Whether the transaction should skip preflight checks. Defaults to `false`
    * @returns {Promise<TransactionSignature>} - The transaction signature
   */
   async sendSmartTransaction(
     instructions: TransactionInstruction[],
     fromKeypair: Keypair,
-    skipPreflightChecks: boolean = true,
-    maxRetries: number = 6,
```

Comment:
You should still have maxRetries imo. This method should provide all the options of regular sendTransaction